### DISCUSSION TITLE
What does the "Advanced Setting / Manual Device” do?

### DISCUSSION THREAD

**Discussion by lychee128** [cagnulein/qdomyos-zwift#311] [Q&A] [answered]
Thanks.

**Comment by cagnulein** [comment, marked as answer]
It forces qZ to connect to that device instead looking for available devices

**Reply by lychee128** [reply]
Is this to select the bike or another device?  I connect to both a bike computer (Flywheel) as well as a heart rate monitor via Bluetooth.  Which one should I select with this function?  Thanks.

**Reply by cagnulein** [reply]
It's to select the bike. You should use your flywheel with this function

**Reply by lychee128** [reply]
Thanks.
